configs/common/modules_lmod.yaml

```diff
@@ -31,6 +31,7 @@ modules:
       - fixesproto
       - flex
       - freetype
+      - gcc-runtime
```

Comment:
This is needed, because `gcc-runtime` is always compiled with `gcc`, no matter which compiler is used